onnxruntime/core/mlas/lib/qgemm_kernel_wasmrelaxedsimd.cpp

```diff
@@ -0,0 +1,568 @@
+/*++
+
+Copyright (c) Microsoft Corporation. All rights reserved.
+
+Licensed under the MIT License.
+
+Module Name:
+
+    qgemm_kernel_wasmsimd.cpp
+
+Abstract:
+
+    This module implements QGEMM kernel for WebAssembly SIMD128.
+
+--*/
+
+#include "mlasi.h"
+#include "qgemm.h"
+
+bool HasUSDot() {
```

Comment:
Does this function work for aarch64 as well? I see only x86 instruction reference below.

Reply:
Currently this solution only works for x64 with `vpdpbusd`, which is " signed and unsigned input", and doesn't work for aarch64.

In V8 on aarch64, `wasm_i32x4_relaxed_dot_i8x16_i7x16_add` will internally be implemented by `SDOT`, which is "signed and signed input", so the `HasUSDot` check will fail.